### ISSUE TITLE
Can provide fake data?

### ISSUE THREAD

**Issue by duyichao** [sheryl-ai/MetaPred#2]
Hello, 
I am a senior. I recently studied your job and it is very interesting, but I have some confusion about the details of the data set. Can you provide **fake data**? 

thank you very much!

**Comment by hiyamgh** [comment]
Yes please or a sample of the data because its important for understanding the problem clearly.

**Comment by hiyamgh** [comment]
Found a repository that base their work on MetaPred (I assume they have the data): https://github.com/AlexKer/federated-metalearning-risk-prediction-ehr
